core/frontend/src/tools/ViewTool.ts

```diff
@@ -3153,8 +3153,8 @@ export class FitViewTool extends ViewTool {
     return EventHandled.No;
   }
 
-  public override onPostInstall() {
-    super.onPostInstall();
+  public override async onPostInstall() {
+    await super.onPostInstall();
```

Comment:
You're my hero Keith!

Can await call to doFit from onPostLocate and remove floating promise ignore.

Reply:
please make that change in this PR

Reply:
done